sig-automation/README.md

```diff
@@ -174,3 +174,24 @@ In terms of known R1 components, this subproject will focus on:
   - Interface configuration (VLAN assignment for example)
   - CNI
 - Multus
+
+### Helm Integration
+**Subproject Lead**: Ciaran Johnston, @ciaranjohnston
+
+This group will study and propose how Helm integrates with Nephio. The goals
+include:
+- Articulate the goals in supporting Helm. For example, is this to primarily
+  to leverage existing investements, or for some additional reasons.
+- Discover vendor, carrier, and other stakeholder requirements
+- Identify and document existing integration points and best practices
+- Identify Nephio changes that are needed to meet the goals and requirements
+
+**Scope**
```

Comment:
This is good as it is. Do we want to state the deliverables of this subproject, such as maybe a doc that states the pros/cons of the different options considered and the rationale for the final recommendation?

Reply:
See latest commit and resolve the conversation if it's OK.

Reply:
Thanks, John.